src/decode.h

```diff
@@ -403,6 +403,7 @@ enum PacketDropReason {
     PKT_DROP_REASON_STREAM_MIDSTREAM,
     PKT_DROP_REASON_NFQ_ERROR,    /**< no nfq verdict, must be error */
     PKT_DROP_REASON_INNER_PACKET, /**< drop issued by inner (tunnel) packet */
+    PKT_DROP_REASON_MAX,
```

Comment:
If the idea is to help users figure out what's happening, wouldn't it make sense to have a drop reason specifically for stream reassembly memcap?

Reply:
yeah agree, but that is outside of the scope of this PR. Can you do a ticket for that?

Reply:
^^ https://redmine.openinfosecfoundation.org/issues/6235